app/helpers/foreman_insights_host_helper.rb

```diff
@@ -0,0 +1,19 @@
+module ForemanInsightsHostHelper
+  def insights_host_overview_buttons(host)
+    search_condition = "hostname=#{host.fqdn}"
+    [
+      {
+        button: link_to_if_authorized(
+          _("Recommendations"),
```

Comment:
how about renaming it to `Insights`?

Reply:
"Insights" has a bit too broad meaning. It's the whole concept of Red Hat looking on your infrastructure. Here we want to be more specific and tell the user that he will be redirected to recommendations about this host.